pages/common/bup.md

```diff
@@ -0,0 +1,23 @@
+# bup
+
+> Very efficient backup system based on the git packfile format.
```

Comment:
What is meant by efficient here? Efficient in terms of time, space used, or something else?

Reply:
Space used, it providing fast incremental saves and global deduplication.

Reply:
Right. Perhaps this would work better then?

```suggestion
> Backup system based on the git packfile format, providing fast incremental saves and global deduplication.
```

Reply:
Yep this work better